content/docs/command-reference/plots/diff.md

```diff
@@ -47,7 +47,21 @@ please see `dvc plots`.
 
 - `--targets <path>` - specific metric files to visualize. These must be listed
   in a [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file) file
-  (see the `--plots` option of `dvc run`).
+  (see the `--plots` option of `dvc run`). When you are specifying multiple
+  arguments for `--targets` and `revisions`, use `--` after all the arguments
+  for `--targets`:
```

Comment:
```suggestion
  (see the `--plots` option of `dvc run`). When specifying multiple targets and
  `revisions`, you may use `--` after this option's arguments, e.g.:
```

Reply:
I'll incorporate these changes in a new PR

Reply:
I see it's done in #1567, thanks.